server/CGameHandler.h

```diff
@@ -345,6 +346,9 @@ class CGameHandler : public IGameCallback, public CBattleInfoCallback, public En
 	void checkVictoryLossConditions(const std::set<PlayerColor> & playerColors);
 	void checkVictoryLossConditionsForAll();
 
+	bool moveArtifactImpl(const ArtifactLocation & al1, const ArtifactLocation & al2, std::vector<MoveArtifact> * artifactsToMove);

```

Comment:
usually Implement postfix means some override over virtual function. If it's not a case, consider another name (maybe moveArtifactInternal and make it private/protected member)

Reply:
Done. It was already private